src/hooks/core/useCoreChatSystemHook.ts

```diff
@@ -24,6 +24,7 @@ import {getAnonymousChatName} from '../../utils/string/StringUtils';
 
 const useCoreChatSystemHook = () => {
   const {localDb, refresh} = useLocalDatabaseHook() as UseLocalDatabaseHook;
+  // TODO: Change this into useUserAuthHook
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The comment suggests that `useProfileHook` should be replaced with `useUserAuthHook`. This change is important for consistency and to ensure that the correct authentication-related data is being used. Here's how you can do it:

```diff
- const {anonProfileId, signedProfileId} = useProfileHook();
+ const {anonProfileId, signedProfileId} = useUserAuthHook();
```
Please make sure to import `useUserAuthHook` at the top of your file if it's not already imported.


<!-- This is an auto-generated comment by OSS CodeRabbit -->